src/lib/attachment-from-file.js

```diff
@@ -16,6 +16,7 @@ function mimeTypeFor(fileName) {
     case 'json': return 'application/json';
     case 'png' : return 'image/png';
     case 'svg' : return 'image/svg+xml';
+    case 'ico' : return 'image/x-icon';
```

Comment:
This section is greatly improved in this not yet merged branch: https://github.com/medic/medic-conf/pull/336/files

Double check that the new way works with `ico` attachments and then revert this change so it doesn't conflict.

Reply:
Done!

Reply:
Fixed!